src/applications/pact-act/containers/results/Results1-1.jsx

```diff
@@ -42,7 +42,9 @@ const Results1Page1 = ({ formResponses, router, viewedIntroPage }) => {
         “presumptive conditions.”
       </p>
       <h2>Exposures related to where you served</h2>
-      <ul>{getDynamicContent(formResponses)}</ul>
+      {getDynamicContent(formResponses)?.length && (
```

Comment:
Results pages 1.1-1.2 used to have dynamic content corresponding to every question that could show the results page. Now that we have a new Burn Pit question that does not have dynamic content, this should be conditionally rendered.